src/data/Philippines-ministry-of-energy_metadata.json

```
@@ -0,0 +1,128 @@
+    "dataSource": {
+      "description": "The Power Development Plan, which is the basis of the scenarios, is available here: https://legacy.doe.gov.ph/pep",
```

The "source" button already appears below here, I don't think we need to add the link in text as well.

```suggestion
      "description": "The Power Development Plan, which is the basis of the scenarios, is available here:",
```